students/ShujinCao/lesson03/assignment/basic_operations.py

```diff
@@ -0,0 +1,59 @@
+"""
+operations
+"""
+import logging
+from customer_model import Customer
+from peewee import fn
+
+def add_customer(customer_id, name, last_name, home_address, phone_number,
+                 email_address, status, credit_limit):
+    try:
+        credit_limit = float(credit_limit)
+    except ValueError as err:
+        logging.error(err)
+        raise
+
+    new_customer = Customer.create(
+        customer_id=customer_id,
+        name=name,
+        last_name=last_name,
+        home_address=home_address,
+        phone_number=phone_number,
+        email_address=email_address,
+        status=status,
+        credit_limit=credit_limit)
+
+    new_customer.save()
+
+def search_customer(customer_id):
+    try:
+        newn_customer = Customer.get(Customer.customer_id == customer_id)
+        cstm_dict = {'name': newn_customer.name,
+                     'last_name': newn_customer.last_name,
+                     'email_address': newn_customer.email_address,
+                     'phone_number': newn_customer.phone_number}
+    except Exception as e:
+        cstm_dict = {}
+        return cstm_dict
+
+    return cstm_dict
```

Comment:
You don't actually need two return statements here. One outside of the try/except block will return either the empty dict or the full one, depending on where it was created.